README.md

```diff
@@ -90,7 +90,13 @@ becomes
 Notice the link `<a href="/fr/menu/">...` directs to the french website.
 
 Even if you are falling back to `default_lang` page, relative links built on the *french* site will
-still link to *french* pages.
+still link to *french* pages.v
+
+#### Relativized Absolute Urls
+If you defined a site `url` in your `_config.yaml`, polyglot will automatically relativize links pointing to your absolute site url. If you don't want them relativized, adding a space explicitly to an href prevents the the absolute url from being relativized.
+
+processed: `href="http://mywebsite.com/about"`
+unprocessed: `href=" http://mywebsite.com/about"`
```

Comment:
Should the space trick work for absolute URLs only? Frankly speaking, I think it is easier to understand in a form of "in case you want to keep the URL as is, just prepend it with a whitespace and polyglot would ignore the link".

If whitespace trick works for all kinds of URLs, then the wording should probably be improved. Currently it sounds like the feature is absolute-URL-only.